2022q3/lsof.adoc

```diff
@@ -1,22 +1,19 @@
 === sysutils/lsof major upgrade
 
-Links
-link:https://github.com/lsof-org/lsof[lsof project repo] URL: link:https://github.com/lsof-org/lsof[https://github.com/lsof-org/lsof] +
+Link: +
+lsof project repo: <https://github.com/lsof-org/lsof>
```

Comment:
Maybe? I'd like a second opinion, in case it breaks tests or a production pipeline stage.

Reply:
I have not check if it break anything yet, but breaking tests would not be a problem: I would simply update them if we found better way of doing things.

As for usefulness, I have some doubts. Please see my comment in the dicussion started in issue #530.